paasta_tools/contrib/check_orphans.py

```diff
@@ -113,12 +149,15 @@ async def gather_files(hosts: Set[str]) -> Dict[str, str]:
 
 
 def read_one_nerve_file(nerve_config: str) -> Set[InstanceTuple]:
-    services = json.loads(nerve_config)["services"]
+    nerve_config = json.loads(nerve_config)
     return {
         InstanceTuple(
-            service["host"], service["port"], service["zk_path"][len(PREFIX) :]
+            host_to_ip(nerve_config["instance_id"], service["host"]),
```

Comment:
Could you please update the PR description explaining why we use `nerve_config["instance_id"]` as a host name.

Reply:
It's this part:

> Instead, we use the name field from the nerve registration, which for PaaSTA instances is the hostname of the PaaSTA host which we can use to fetch the configured nerve registrations.

I'll make it more explicit that those are the same thing.

Reply:
The code above uses `instance_data["name"]`, which is obviously the name field.
But here it's `nerve_config["instance_id"]`.  Is it the name field as well?  Why it's called `instance_id` then, if it's "name"? And etc. 🙂  
This is confusing.
Without an additional explanation I'm reading this code like we're trying to resolve the instance name as a host name.

Reply:
I agree that the naming is confusing but that comes from what nerve does): https://github.com/yelp/nerve#configuration

I'll add a comment in the code in addition to the PR description to clarify.